nekoyume/Assets/_Scripts/Analyzer.cs

```diff
@@ -12,6 +12,8 @@ public class Analyzer
 
         private readonly MixpanelValueFactory _mixpanelValueFactory;
 
+        public int GuideQuestStageId { get; set; }
+
```

Comment:
This variable seems not fit with the `Analyzer`. Can we pair about this?

Reply:
OK I'll solve another problem and give you a mention. thx